### PR TITLE
feat: Dark mode improvements

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

Fixes https://linear.app/chatwoot/issue/CW-2336/dark-mode-issues

This PR includes,

1. Fixes banner button and color issues.
2. Fixes order list is not showing in message bubbles.
3. Fixes issue with private note bg color and text color.
4. Private note editor text color issue.
5. Fixes gradient issue for attachment bubbles that cause timestamp is not visible with which images

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality not to work as expected)

**Screenshots**
https://linear.app/chatwoot/issue/CW-2336#comment-e62ab146

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/App.vue [hunk 1]
```diff
@@ -112,9 +112,11 @@ export default {
       ) {
         this.theme = 'dark';
         document.body.classList.add('dark');
+        document.documentElement.setAttribute('style', 'color-scheme: dark;');
```
Comment (pranavrajs) [inline app/javascript/dashboard/App.vue:115]: @iamsivin Can you please share why this is needed.

File: app/javascript/dashboard/assets/scss/widgets/_conversation-view.scss [hunk 2]
```diff
@@ -28,7 +28,12 @@
     }
 
     &::before {
-      @apply bg-gradient-to-t from-transparent via-transparent to-slate-800 bg-repeat bg-clip-border bottom-0 h-[20%] content-[''] left-0 absolute w-full opacity-80;
+      background-image: linear-gradient(
```
Comment (pranavrajs) [inline app/javascript/dashboard/assets/scss/widgets/_conversation-view.scss:31]: Can we use the tailwind helper without writing a custom CSS here?

**Review by pranavrajs — CHANGES_REQUESTED** [review]
Minor comments, please take a look.

File: app/javascript/dashboard/App.vue (hunk 1 above)
Comment (iamsivin) [reply app/javascript/dashboard/App.vue:115]: @pranavrajs With this, the browser can apply its default dark or light styling to certain elements, such as scroll bars, check boxes, and text areas.

File: app/javascript/dashboard/assets/scss/widgets/_conversation-view.scss (hunk 2 above)
Comment (iamsivin) [reply app/javascript/dashboard/assets/scss/widgets/_conversation-view.scss]: @pranavrajs Updated the PR https://github.com/chatwoot/chatwoot/pull/7658/commits/1f5b3fb9b62a5bccd2eebe5e64bacae30ee9adff

**Review by nithindavid — APPROVED** [review]

File: app/javascript/dashboard/App.vue (hunk 1 above)
Comment (iamsivin) [reply app/javascript/dashboard/App.vue:115]: For example, here are some screenshots,

1. **Audio player**
image

2. **Scrollbar**

| **Darkmode**   | **Lightmode**   |
|------------|------------|
| image   | image   | 

3. **Checkbox**
image

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**